src/NHibernate/Criterion/QueryOver.cs

```diff
@@ -102,10 +102,10 @@ private IFutureValue<U> FutureValue<U>()
 			return criteria.FutureValue<U>();
 		}
 
-		/// <summary>
-		/// Get an executable instance of <c>IQueryOver&lt;TRoot&gt;</c>,
-		/// to actually run the query.</summary>
-		public IQueryOver<TRoot,TRoot> GetExecutableQueryOver(ISession session)
+        /// <summary>
+        /// Get an executable instance of <c>IQueryOver&lt;TRoot&gt;</c>,
+        /// to actually run the query.</summary>
+        public IQueryOver<TRoot,TRoot> GetExecutableQueryOver(ISession session)
```

Comment:
Whitespace miss probably introduced during rebasing. I am anyway rebasing again, while rewording commits for including Jira number.